doc/CamOverlayPainter.md

```diff
@@ -0,0 +1,116 @@
+# CamOverlayDrawingAPI
+
+Three modules for even easier control of CamOverlay drawing API.
+
+## ResourceManager
+
+-   Downloads and stores resources.
+
+-   **registerImage(moniker: string, fileName: string), registerFont(moniker: string, fileName: string)** - Adds the resource specified by `fileName` to this `ResourceManager`.
+
+-   **async image(co: CamOverlayDrawingAPI, moniker: string), async font(co: CamOverlayDrawingAPI, moniker: string)** - Uploads the resource specified by `moniker` to the camera, if it hasn't been done already. Returns the data, which that camera sent as a response.
+
+-   **clear()** - Removes all data from ResourceManager. However, all resorces will be still registered.
+
+## Frame
+
+-   **Frame(options: FrameOptions, rm: ResourceManager, customDraw?: DrawingCallback)**
+
+    ```javascript
```

Comment:
tady by mel byt typescript, takhle ti to zmenilo stredniky na carky coz je vlastne spatne, zkontroluj prosim i ostatni vyskyty..

Reply:
Myslím, že nikde jinde s tím nebyl problém.